semver/mod.ts

```diff
@@ -542,11 +542,14 @@ export class SemVer {
     this.format();
   }
 
-  format(): string {
+  format({ includeBuild } = { includeBuild: false }): string {
```

Comment:
Should this be optional? How about showing the build metadata always?

Reply:
We cannot do that, as `format()` call is used for semver comparison, ref: https://github.com/denoland/deno_std/issues/3110#issuecomment-1383221174

Also:
> Although format itself is not pure function, so maybe an alternative just for printing would be preferable.